packages/@okta/vuepress-site/docs/concepts/user-profiles/index.md

```diff
@@ -0,0 +1,78 @@
+
+#### Group profile type
+
+Okta groups simplify management of multiple users of the same type, see [About groups](https://help.okta.com/en/prod/Content/Topics/users-groups-profiles/usgp-about-groups.htm). The Group profile itself consists of attributes and can be defined and managed with the [Groups API](/docs/reference/api/groups/). See the [Group object](/docs/reference/api/groups/#group-object) and [Group attributes](/docs/reference/api/groups/#group-attributes) for further information.
+
+#### Custom profile type
+
+The custom user profile type is based on the Okta user profile type and is used to define different types of users, for example administrators, contractors, help desk, and so on. Similar to the default Okta profile, the custom user profile type contains 31 attributes and can be extended with custom attributes. See also [About custom user types in Universal Directory](https://help.okta.com/en/prod/Content/Topics/users-groups-profiles/usgp-usertypes-about.htm).
+
+The [User Types API](/docs/reference/api/user-types/) defines and manages the custom profile types.
+
+User profiles can only belong to one user profile type. The [User object property](/docs/reference/api/users/#user-properties) `type` defines the custom user profile (or default profile) the user is associated with.
+
+>**Note:** The default [Profile object property](/docs/reference/api/users/#default-profile-properties) `userType` is a user profile attribute and is not a reference to the default or custom profile type.
+
+### App user profile type
+
+The app user profile type defines the attributes available for a user of that application in the Universal Directory. The app user profile attributes are mapped to the user profile and determines the data that can be sent to or imported from an app. Similar to user profiles, the app user profile has base attributes and custom attributes. The available custom attributes, however, are determined by the application. You can manage the app user profile type with the [Apps API](/docs/reference/api/apps/). Review the [Application User Profile](/docs/reference/api/apps/#application-user-profile-object) object and the [Application User](/docs/reference/api/apps/#application-user-object) object for further details.
+
+## Universal Directory Schemas
+
+Many different types of data can be stored in a user profile: strings, numbers, dates, lists, and so on. A schema is a description of what type of information is stored in a user profile. Each element in a schema is known as an “attribute” and each attribute has the following metadata, or properties:
```

Comment:
Style guide says: "Begin each item in a list with a capital letter unless there's a reason not to (for example, it’s a command that's always lowercase). 
strings > Strings

Reply:
Is this for ordered and unordered lists, though? Maybe I can just recast sentence. How 'bout: "Many different types of data can be stored in a user profile such as string, numbers, dates, lists, and so on."